static/i18n/en/app.json

```diff
@@ -265,11 +265,7 @@
         ]
       },
       "changelly": {
-        "bullets": [
-          "BTC, ETH, USDT and 50+ others",
-          "Multi-currency",
-          "Centralized"
-        ]
+        "bullets": ["BTC, ETH, USDT and 50+ others", "Multi-currency", "Centralized"]
```

Comment:
Is this even still used? I believe it belongs to the deprecated provider selector screen, can likely be killed completely

Reply:
This change is the noise introduced by my code editor that format files on saving, I don't touch anything here, my bad.
I fast-checked the code and I can confirm that these lines seem not to be used. I can remove them if you want or just wait that the rework on swap v2 is ready to merge then remove all translations from the v1 swap key.
That will allow us to remove the 200+ translations from the legacy swap feature and rename "swap2" to "swap".

Tell me what do you prefer.

Reply:
we need to keep v1 to be able to switch between them until it's stable, but these ones in particular can die, please

Reply:
Removing them [here](https://github.com/juan-cortes/ledger-live-desktop/pull/13) 🛩️